### ISSUE TITLE
[react-ui-validations] не используется horizontalOffset

### ISSUE THREAD

**Issue by slawwan** [skbkontur/retail-ui#1251]
сейчас ValidationContext поддерживает horizontalOffset но ValidationContainer его не пробрасывает

**Comment by dzekh** [comment]
Из описания понятно чуть больше чем ничего. Это про прокрутку формы до нужного места, при валидации по сабмиту, учитывая что контейнер перекрыт какой-то залипашкой?

А есть скриншоты может? 
Если мы выкинем неиспользуемый код, можно ли будет реализовать прокрутку в этих случаях?

**Comment by wKich** [comment]
@dzekh, да, ты всё верно понял. У валидаций можно задать отступ, который будет применяться при прокрутке к элементу на случай, если в форме есть залипающие элементы.

**Comment by dzekh** [comment]
Ну в общем нужно чтоб это работало.

**Comment by wKich** [comment]
Думаю что нет
